packages/css/search.css

```
@@ -1,43 +1,43 @@
 @layer fds.search {
-  .formField {
+  .fds-search__container {
```

Looks like you forgot to change this after changing the classname in the React component.
```suggestion
  .fds-search {
```